editor.planx.uk/src/@planx/components/NumberInput/Editor.tsx

```diff
@@ -83,14 +80,30 @@ export default function NumberInputComponent(props: Props): FCReturn {
                   onChange={() =>
                     formik.setFieldValue(
                       "allowNegatives",
-                      !formik.values.allowNegatives,
+                      !formik.values.allowNegatives
                     )
                   }
                 />
               }
               label="Allow negative numbers to be input"
             />
           </InputRow>
+          <InputRow>
+            <FormControlLabel
+              control={
+                <Switch
+                  checked={formik.values.onlyWholeNumbers}
+                  onChange={() =>
+                    formik.setFieldValue(
+                      "onlyWholeNumbers",
```

Comment:
Not wedded to the variable name or way I'm framing this option. Would appreciate alternative suggestions